software/fishstick-stm32f401/src/main.rs

```diff
@@ -0,0 +1,38 @@
+#![deny(unsafe_code)]
+#![allow(clippy::empty_loop)]
```

Comment:
Is this clippy pragma used anywhere? Also I don't see `clippy` listed as a dependency anywhere -- how does it get installed?

Reply:
removed. probably from some copy paste i did